libs/domain/user/address-form/src/address-form.component.ts

```diff
@@ -54,6 +58,23 @@ export class AddressFormComponent
 
   @property({ type: Boolean }) enableDefaultShipping?: boolean;
   @property({ type: Boolean }) enableDefaultBilling?: boolean;
+  @property({ type: Object }) set address(value: Address) {
+    // TODO: introduce generic method to apply form data on form element
```

Comment:
do you want to keep it?

Reply:
Tobias Ouwejan
  [4 minutes ago](https://spryker.slack.com/archives/D02DE33H509/p1684314547130819?thread_ts=1684314538.271339&cid=D02DE33H509)
I'm in favor of dropping it now

Tobias Ouwejan
  [4 minutes ago](https://spryker.slack.com/archives/D02DE33H509/p1684314564651039?thread_ts=1684314538.271339&cid=D02DE33H509)
only when we have refelcction of form data using form component (or something else), we can bring it back

Tobias Ouwejan
  [3 minutes ago](https://spryker.slack.com/archives/D02DE33H509/p1684314591497099?thread_ts=1684314538.271339&cid=D02DE33H509)
it woudl mean that for now we'd not have address form data being restored from memory, which is okish